docs/examples/vector_stores/QdrantIndexDemo.ipynb

```diff
@@ -32,7 +32,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        #location=":memory:"
</pre><p>do we need to keep this line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10786/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I would. It shows how to store data in memory with the goal to query it right away. So, for didactic purposes, it's useful.

Naturally, the user needs to uncomment it and instead to comment the other one (path=path), as they're mutually exclusive.